microsoft/PowerToys
# VSCode to full name, disable Visual Studio Code plugin by default

## Summary of the Pull Request

**What is this about:**
![image](https://user-images.githubusercontent.com/17161067/110308180-7d95e000-8008-11eb-884e-5566c81f19c9.png)

**What is include in the PR:** 

**How does someone test / validate:** 
- delete `PowerToys Run\settings.json` file
- start PowerToys
- Check that the Visual Studio Code Workspaces plugin is disabled

## Quality Checklist

- [X] **Linked issue:** #10067

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
